qemu_v8.mk

```diff
@@ -189,6 +298,68 @@ soc-term:
 soc-term-clean:
 	$(MAKE) -C $(SOC_TERM_PATH) clean
 
+################################################################################
+# mkimage - create images to be loaded by U-Boot
+################################################################################
+# Without the objcopy, the uImage will be 10x bigger.
+$(KERNEL_UIMAGE): u-boot linux | $(BINARIES_PATH)
+	${AARCH64_CROSS_COMPILE}objcopy -O binary \
+					-R .note \
+					-R .comment \
+					-S $(LINUX_PATH)/vmlinux \
+					$(BINARIES_PATH)/linux.bin
+	$(MKIMAGE_PATH)/mkimage -A arm64 \
+				-O linux \
+				-T kernel \
+				-C none \
+				-a $(KERNEL_LOADADDR) \
+				-e $(KERNEL_ENTRY) \
+				-n "Linux kernel" \
+				-d $(BINARIES_PATH)/linux.bin $(KERNEL_UIMAGE)
+
+$(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)
+	ln -sf $(ROOT)/out-br/images/rootfs.cpio.gz $(BINARIES_PATH)
+	$(MKIMAGE_PATH)/mkimage -A arm64 \
+				-T ramdisk \
+				-C gzip \
+				-a $(ROOTFS_LOADADDR) \
+				-e $(ROOTFS_ENTRY) \
+				-n "Root file system" \
+				-d $(ROOTFS_GZ) $(ROOTFS_UGZ)
+
+################################################################################
+# XEN
+################################################################################
+.PHONY: xen
+xen-defconfig:
+	$(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
+
+xen-common: xen-defconfig
+	cd $(XEN_PATH)/xen && \
+	tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf
+
+xen: xen-common | $(BINARIES_PATH)
+	$(MAKE) -C $(XEN_PATH) dist-xen \
+	XEN_TARGET_ARCH=arm64 \
+	CONFIG_XEN_INSTALL_SUFFIX=.gz	\
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	ln -sf $(XEN_PATH)/xen $(BINARIES_PATH)
+
+XEN_TMP ?= $(BINARIES_PATH)/xen_files
+
+$(XEN_TMP):
+	mkdir -p @
```

Comment:
`$@`

Reply:
Sorry, I had this change done locally but forgot to add it in the patch. Will correct it.